src/sparky/sparky_chain.ts

```diff
@@ -37,7 +37,7 @@ export function sparkyChain(log: FuseBoxLogAdapter): ISparkyChain {
     }
 
     if (newLocation && newLocationBase) {
-      const root = ensureAbsolutePath(newLocation, env.APP_ROOT);
+      const root = ensureAbsolutePath(newLocation, env.SCRIPT_PATH);
```

Comment:
Yeah I think that's what It should be.. I never had problems cuz fuse.ts is usually at the root ;-)